src/main/java/org/elasticsearch/search/aggregations/bucket/geogrid/InternalGeoHashGrid.java

```diff
@@ -157,7 +156,7 @@ public void writeTo(StreamOutput out) throws IOException {
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             builder.startObject();
-            builder.field(CommonFields.KEY, getKeyAsText());
+            builder.field(CommonFields.KEY, new StringText(getKeyAsString()));
```

Comment:
Do we actually need to wrap into a Text object here, it should work with the string?